Dockerfile

```diff
@@ -0,0 +1,10 @@
+# This image will be published as dspace/dspace-angular
+# See https://dspace-labs.github.io/DSpace-Docker-Images/ for usage details
+
+FROM node:8
+WORKDIR /app
+ADD . /app/
```

Comment:
Once I am able to get a static version of this image to behave properly, I would like to also support this functionality.  It might be as simple as providing a volume to replace the /app directory.